### PR TITLE
EX-3637 Prebid should not pass requests without mime types

### PR DESCRIPTION
**Pull request by mikhalovich**

### CODE REVIEW

File: modules/sovrnBidAdapter.js [hunk 1]
```diff
@@ -40,12 +40,17 @@ export const spec = {
    * @param {object} bid the Sovrn bid to validate
    * @return boolean for whether or not a bid is valid
    */
-  isBidRequestValid: function(bid) {
+  isBidRequestValid: function c(bid) {
```
Comment (cpabst) [inline modules/sovrnBidAdapter.js]: What's going on with these 2 lines?

**Review by cpabst — COMMENTED** [review]
A question

**Review by jrosendahl — CHANGES_REQUESTED** [review]
Please see updated ticket

File: modules/sovrnBidAdapter.js (hunk 1 above)
Comment (mikhalovich) [reply modules/sovrnBidAdapter.js]: Thanks, Chris. I didn't notice it. I removed it

**Comment by mikhalovich** [comment]
@jrosendahl Done

**Review by cpabst — APPROVED** [review]
LGTM

**Review by jrosendahl — APPROVED** [review]
lgtm

**Comment by maximpakhomov** [comment]
@jrosendahl @cpabst if this PR is no longer needed, can it be closed or merged?

**Comment by cpabst** [comment]
Yes, this should be closed.

**Pull request closed without merging**